job/models.py

```diff
@@ -216,19 +218,27 @@ class TIME_STEP_CHOICES(models.IntegerChoices):
         help_text=("The threshold for the difference between the solution's objective value and the best possible "
                    "value at which the solver terminates")
     )
+
     add_soc_incentive = models.BooleanField(
         default=True,
         blank=True,
         help_text=("If True, then a small incentive to keep the battery's state of charge high is added to the "
                    "objective of the optimization.")
     )
+
     run_bau = models.BooleanField(
-        default=True,
         blank=True,
+        null=True,
         help_text=("If True then the Business-As-Usual scenario is also solved to provide additional outputs such as "
                    "the LCC and BAU costs.")
```

Comment:
Should this be "such as the NPV and BAU costs" ?

Reply:
Updated in https://github.com/NREL/REopt_API/pull/328/commits/2c3e977834c2671ce76fcfdd32faed8dbe91d7f6